andoriyaprashant/DevDocsHub
# Refactor: Moved Inline CSS to External Stylesheets for Improved Performance

**Description:**
This PR addresses the performance issue caused by using inline CSS, which can slow down the webpage during the loading phase. By moving these styles to external stylesheets, we improve caching, reduce HTML file size, and enhance overall page load speed.

**Issue Number**
#42 

**Problem:**
Inline CSS was being used in multiple places across the application, increasing the size of the HTML and preventing proper browser caching.
This practice negatively impacts the load time, especially on slower networks or with repeat visits, as the inline styles are reloaded with each page request.
**Solution:**
I have refactored all inline styles and moved them into the appropriate external CSS files.
This reduces the amount of inline code, improves caching efficiency, and optimizes page loading performance.
**Benefits:**
Improved loading time due to reduced HTML size and better browser caching.
Separation of concerns, which makes the codebase cleaner and easier to maintain.
Better SEO and accessibility since performance improvements also positively impact user experience and page rankings.
**Steps to Test:**
Load the pages where inline styles were previously used.
Verify that the styles are still applied correctly but are now sourced from external stylesheets.
Check the performance (load time) before and after the changes using browser developer tools (e.g., Lighthouse in Chrome).
